jung-algorithms/src/test/java/edu/uci/ics/jung/layout/spatial/FRLayoutsTest.java

```diff
@@ -0,0 +1,159 @@
+package edu.uci.ics.jung.layout.spatial;
+
+import com.google.common.collect.Maps;
+import com.google.common.graph.GraphBuilder;
+import com.google.common.graph.MutableGraph;
+import edu.uci.ics.jung.layout.algorithms.FRBHLayoutAlgorithm;
+import edu.uci.ics.jung.layout.algorithms.FRBHVisitorLayoutAlgorithm;
+import edu.uci.ics.jung.layout.algorithms.FRLayoutAlgorithm;
+import edu.uci.ics.jung.layout.algorithms.IterativeLayoutAlgorithm;
+import edu.uci.ics.jung.layout.algorithms.LayoutAlgorithm;
+import edu.uci.ics.jung.layout.model.LayoutModel;
+import edu.uci.ics.jung.layout.model.LoadingCacheLayoutModel;
+import edu.uci.ics.jung.layout.model.Point;
+import java.util.Map;
+import org.junit.AfterClass;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * This test makes a very small graph, sets initial locations for each node, and each time, it runs
+ * a version of FRLayout. FRLayoutAlgorithm - the JUNG legacy version FRBHLayoutAlgorithm - modified
```

Comment:
nit: suggest using HTML list to make this more readable:
```
<ul>
<li>FRLayoutAlgorithm: the JUNG legacy version
<li>FRBHLayoutAlgorithm: modified to use...
...
</ul>
```

Reply:
Agreed. I suspect that all of our javadoc could use some grooming (and corrections!)

Reply:
very true :)